vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    state: present
+
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
+    home: /home/minio
+
+- name: create minio bin directory
+  become: yes
+  file:
+    path: "{{root_folder}}/app/minio/{{ item }}"
+    state: directory
+    owner: minio
+  with_items:
+    - "bin"
+    - "data"
```

Comment:
No need for data directory here. Create this directory in  {{root_folder}}/data/minio/

Reply:
That being said you don't really need bin directory, because {{root_folder}}/app/minio/ should be the bin directory.

Reply:
> No need for data directory here. Create this directory in {{root_folder}}/data/minio/
All the created Volumes are going to be located to  **{{root_folder}}/app/minio/data**  so Minio need a Data Directory


> That being said you don't really need bin directory, because {{root_folder}}/app/minio/ should be the bin directory.
The Minio Executable file is located in **{{root_folder}}/app/minio/bin**

Reply:
changed the folder structure as per the suggestion